services/clusters-keeper/src/simcore_service_clusters_keeper/modules/clusters.py

```diff
@@ -69,10 +71,14 @@ async def create_cluster(
         tags=creation_ec2_tags(app_settings, user_id=user_id, wallet_id=wallet_id),
         startup_script=create_startup_script(
             app_settings,
-            get_cluster_name(
+            cluster_machines_name_prefix=get_cluster_name(
                 app_settings, user_id=user_id, wallet_id=wallet_id, is_manager=False
             ),
-            ec2_instance_boot_specs,
+            ec2_boot_specific=ec2_instance_boot_specs,
+            additional_custom_tags={
+                AWSTagKey("user_id"): AWSTagValue(f"{user_id}"),
+                AWSTagKey("wallet_id"): AWSTagValue(f"{wallet_id}"),
```

Comment:
Do I understand correctly, that this means the worker nodes will also be detected [here](https://git.speag.com/oSparc/e2e-testing/-/blob/production_zmt_aws/computational_backend/osparc-aws/osparc_aws/osparc_aws.py?ref_type=heads#L70) (in the e2e tests for the computational backend)

Reply:
yes correct